e2e-tests/cypress/tests/integration/channels/notifications/reset_notification_when_settings_canceled_spec.js

```diff
@@ -0,0 +1,125 @@
+        cy.shouldHavePluginUploadEnabled();
+        cy.apiUploadAndEnablePlugin(callsPlugin);
+
+        cy.apiInitSetup().then(({team, user, channel}) => {
+            // # Login as user and visit channel
+            cy.apiLogin(user);
+            cy.visit(`/${team.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-53166 Notification sound modal selection should reset when settings canceled', () => {
+        // # Call function that clicks on Settings -> Notifications -> Desktop Notifications -> Notification sound -> Change sound -> Cancel -> Desktop Notifications
+        openSettingsAndChangeNotification('desktopNotification');
+
+        // # Call function that clicks on Settings -> Notifications -> Desktop Notifications -> Notification sound for incoming calls -> Cancel -> Desktop Notifications
+        openSettingsAndChangeNotification('callsDesktopSound');
+    });
+
+    function openSettingsAndChangeNotification(type) {
+        // # Open 'Settings' modal
+        cy.uiOpenSettingsModal().within(() => {
+            // # Navigate to Desktop Notification Settings
+            navigateToDesktopNotificationSettings(type);
+
+            // # Change Notification selection
+            setNotificationToDown(type);
+
+            // # Click Cancel button
+            cy.uiCancelButton().click();
+
+            // # Navigate to Desktop Notification Settings
+            navigateToDesktopNotificationSettings(type);
+        });
+    }
+
+    function setNotificationToDown(type) {
+        switch (type) {
+        case 'desktopNotification':
+            // # Change Notification sound selection value is set to Down
+            cy.get('#displaySoundNotification').
+                find('.react-select__dropdown-indicator').
+                click().
+                get('.react-select__menu').
+                contains('Down').
+                click();
+            break;
+        case 'callsDesktopSound':
+            // # Change Notification Notification sound for incoming calls selection value is set to Down
+            cy.get('#displayCallsSoundNotification').
+                find('.react-select__dropdown-indicator').
+                click().
+                get('.react-select__menu').
+                contains('Down').
+                click();
+            break;
+        default:
+            break;
+        }
+
+        // * Verify Notification display changed to Down
+        verifyNotificationSelectionValue(type, 'Down');
+    }
+
+    function navigateToDesktopNotificationSettings(type) {
+        // # Click on the 'Edit' button next to Desktop Notifications
+        cy.get('#desktopEdit').should('be.visible').click();
+
+        // * Verify that the Notification is set to On
+        verifyNotificationIsOn(type);
+
+        // * Verify Notification selection display default value (Bing)
+        verifyNotificationSelectionValue(type, 'Bing');
+    }
+
+    function verifyNotificationIsOn(type) {
```

Comment:
Hi @dhnlr 
just curious if we need this step. They are by default on, when the plugin is enabled. (is that not the case?)

better to have an assertion on `#displaySoundNotification` and `#displayCallsSoundNotification` fields

Reply:
Hi, @yasserfaraazkhan. 
I don't know it will be on by default. Yeah better assert `#displaySoundNotification` and `#displayCallsSoundNotification`.